.editorconfig

```diff
@@ -0,0 +1,5 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
```

Comment:
https://editorconfig.org/ configures settings that will apply to multiple text editors just for this repository.

- `root=true` means that `.editorconfig` capable text-editors (see link earlier) won't go looking for configuration from top leve l directories.
- line endings are enforced to be `lf`, which the editors should detect automatically though, as a pair to `.gitattributes` `eol=lf` and to workaround https://github.com/thlorenz/doctoc/issues/161 just in case anyone uses Windows.
- well I certainly hope everyone uses utf-8